README.md

```diff
@@ -74,3 +74,32 @@ cd new-site
 npm install
 npm start
 ```
+
+## CONTRIBUTING
+
+### Setting Up a Local Copy
+
+1. Clone the repo with `git clone https://github.com/h5bp/create-html5-boilerplate.git`
+2. Run `npm install` in the root `create-html5-boilerplate` folder.
+3. run `npm link` to make npm run local copy instead of downloading from registry
+4. now you can use `npm init html5-boilerplate` and `npx create-html5-boilerplate`
```

Comment:
minor nitpick - suggest starting each list item with a capital letter for consistency

Reply:
Fixed, thanks :)